packages/okta-angular/src/okta/okta.guard.ts

```diff
@@ -38,15 +39,29 @@ export class OktaAuthGuard implements CanActivate {
       return true;
     }
 
+    /**
+     * Get the operation to perform on failed authentication from
+     * either the global config or route data injection.
+     */
+    const authGuardRedirect = route.data['authGuardRedirect'] || this.oktaAuth.getOnAuthRequired();
```

Comment:
`authGuardRedirect` should probably be consistent in the places we allow extensibility (`onAuthRequired` or something)

Reply:
Agreed that we should consistently use `onAuthRequired`